### ISSUE TITLE
connection.Close() crashes on Universal Windows

### ISSUE THREAD

**Issue by Maarten88** [NVentimiglia/Websockets.PCL#6]
I tried to close the connection, but I found this function immediately crashes with:

```
A method was called at an unexpected time. (Exception from HRESULT: 0x8000000E)
```

I confirmed this by adding the statement connection.Close() near the bottom of DoTest() in TestSample.cs

It is caused by these lines in the UWP version of WebsocketConnection.cs:

```
_websocket.Closed -= _websocket_Closed;
_websocket.MessageReceived -= _websocket_MessageReceived;
```

If I comment these lines out, I can close the connection, but maybe that'll result in a memory leak, which is exactly what I was trying to avoid by calling Close...

**Comment by NVentimiglia** [comment]
I have replication, will push a fix

**Comment by NVentimiglia** [comment]
Fixed in 1.1.9
